docs/faq.rst

```diff
@@ -119,8 +119,11 @@ adjust the level or algorithm).
 levels. Which of them is the best for you pretty much depends on your
 use case, your data, your hardware -- so you need to do an informed
 decision about whether you want to use compression, which algorithm
-and which level you want to use. This is why compression defaults to
-none.
+and which level you want to use.
```

Comment:
the previous sentence is also somehow related to have a "none" default.
instead we should say lz4 is default because it is always better than none,
but that if one needs a better compression, one needs to specify it.